doc/reference/configuration/configuration_reference.rst

```diff
@@ -2389,3 +2389,350 @@ The ``security`` section defines configuration parameters related to various sec
     | Type: boolean
     | Default: false
     | Environment variable: TT_SECURITY_SECURE_ERASING
+
+
+
+
+..  _configuration_reference_sharding:
+
+sharding
+--------
+
+The ``sharding`` section defines configuration parameters related to sharding.
```

Comment:
The link here might be useful:
https://docs.d.tarantool.io/en/doc/sharding-reference/concepts/sharding/#sharding

Reply:
Added a link to https://docs.d.tarantool.io/en/doc/sharding-reference/book/admin/vshard_admin/ - I think, this will be the main topic related to sharding configuration after v3.1 is released.